tests/test_hotjar.py

```
@@ -1,12 +1,10 @@
 from typing import Any, Dict
 
-
-from platzky.platzky import create_app_from_config, Config
+from platzky.platzky import Config, create_app_from_config
 
 
 def test_that_plugin_loads_hotjar():
-
-    secret_id_for_testing = "super_secret_id"
+    secret_id_for_testing = "1234567"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "secret_id_for_testing"

(S105)